src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Performance/PreferLengthOverAnyTests.cs

```diff
@@ -254,5 +255,70 @@ public class MyCollection {
 
             return VerifyCS.VerifyAnalyzerAsync(code);
         }
+
+        [Fact, WorkItem(7063, "https://github.com/dotnet/roslyn-analyzers/issues/7063")]
+        public Task WhenInExpressionTree_NoDiagnostic()
+        {
+            const string code = """
+                                using System;
+                                using System.Collections.Generic;
+                                using System.Linq;
+                                using System.Linq.Expressions;
+
+                                public class Tests {
+                                    public void M() {
+                                        var array = new int[0];
+                                        Evaluate(() => array.Any());
+                                    }
+                                
+                                    private void Evaluate(Expression<Func<bool>> expression)
+                                    {
+                                    }
+                                }
+                                """;
```

Comment:
💡 The indentation here would be vastly improved if the closing `"""` was indented exactly 4 spaces relative to `const`

Reply:
Like this?